### PR TITLE
Announce 2022 Keynote Presenter: Sumana Harihareswara

### PR DESCRIPTION
**Pull request by nhandler**
This change adds a new blog post (and image) announcing Sumana Harihareswara as one of our 2022 Keynote Presenters.

### CODE REVIEW

File: _posts/2022-09-28-keynote-sumana-harihareswara.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+[Sumana Harihareswara](https://www.harihareswara.net/) is an open source
+contributor and leader who has contributed to pip, GNOME, MediaWiki,
+Dreamwidth, GNU Mailman, and other open source projects -- and is
+working on a book to teach what she's learned along the way. She has
+keynoted LibrePlanet and other open source conventions, spoken at PyCon
+and OSCON, performed standup comedy and theater at several tech
+conferences, and manages and maintains open source projects as
+[Changeset Consulting](http://changeset.nyc/).  Her work has earned her
+an Open Source Citizen Award and a Google Open Source Peer Bonus. She
+lives in New York City
```
Comment (lisushka) [inline _posts/2022-09-28-keynote-sumana-harihareswara.md:26]: Need a full stop here.

File: _posts/2022-09-28-keynote-sumana-harihareswara.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Image Credits [Alison Wilgus](http://www.tcj.com/alison-wilgus-day-three/)
+
+[Sumana Harihareswara](https://www.harihareswara.net/) is an open source
+contributor and leader who has contributed to pip, GNOME, MediaWiki,
+Dreamwidth, GNU Mailman, and other open source projects -- and is
+working on a book to teach what she's learned along the way. She has
+keynoted LibrePlanet and other open source conventions, spoken at PyCon
+and OSCON, performed standup comedy and theater at several tech
+conferences, and manages and maintains open source projects as
```
Comment (lisushka) [inline _posts/2022-09-28-keynote-sumana-harihareswara.md:23]: The tense switch here is a little weird - how do we feel about making a slight edit?

File: _posts/2022-09-28-keynote-sumana-harihareswara.md (hunk 1 above)
Comment (nhandler) [reply _posts/2022-09-28-keynote-sumana-harihareswara.md:26]:
```suggestion
lives in New York City.
```

File: _posts/2022-09-28-keynote-sumana-harihareswara.md (hunk 2 above)
Comment (nhandler) [reply _posts/2022-09-28-keynote-sumana-harihareswara.md:23]: I'm tempted to leave it. While not perfect, I think it is still fully comprehensible. There are also other grammar changes we would likely want to start making, and at that point. It is probably easier to just fallback on, we publish what the speaker provides. If we want, we can add a line saying someting like, "In her own words..."

**Review by lisushka — APPROVED** [review]
Let's do it.

File: _posts/2022-09-28-keynote-sumana-harihareswara.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+published: true
+categories: news
+tags: '2022'
+---
+
+We are excited to be able to announce the first of our amazing keynote presenters for SeaGL 2022: Sumana Harihareswara!
+
+![Sumana Harihareswara](/img/posts/2022_Keynote_Sumana.jpg)
+
+Image Credits [Alison Wilgus](http://www.tcj.com/alison-wilgus-day-three/)
```
Comment (AndrewKvalheim) [inline _posts/2022-09-28-keynote-sumana-harihareswara.md:15]: This could use `https`.

Comment (nhandler) [reply _posts/2022-09-28-keynote-sumana-harihareswara.md:15]:
```suggestion
Image Credits [Alison Wilgus](https://www.tcj.com/alison-wilgus-day-three/)
```

File: _posts/2022-09-28-keynote-sumana-harihareswara.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+[Sumana Harihareswara](https://www.harihareswara.net/) is an open source
+contributor and leader who has contributed to pip, GNOME, MediaWiki,
+Dreamwidth, GNU Mailman, and other open source projects -- and is
+working on a book to teach what she's learned along the way. She has
+keynoted LibrePlanet and other open source conventions, spoken at PyCon
+and OSCON, performed standup comedy and theater at several tech
+conferences, and manages and maintains open source projects as
+[Changeset Consulting](http://changeset.nyc/).  Her work has earned her
```
Comment (nhandler) [inline _posts/2022-09-28-keynote-sumana-harihareswara.md:24]:
```suggestion
[Changeset Consulting](https://changeset.nyc/).  Her work has earned her
```

**Pull request merged**